starknet_devnet/util.py

```diff
@@ -66,6 +69,20 @@ def fixed_length_hex(arg: int) -> str:
 #     # otherwise a URL; perhaps check validity
 #     return name
 
+class DumpOn(Enum):
+    """Enumerate possible dumping frequencies."""
+    EXIT = auto()
+    TRANSACTION = auto()
+
+DUMP_ON_OPTIONS = [e.name.lower() for e in DumpOn]
+DUMP_ON_OPTIONS_STRINGIFIED = ", ".join(DUMP_ON_OPTIONS)
+
+def parse_dump_on(option: str):
+    """Parse dumping frequency option."""
+    if option in DUMP_ON_OPTIONS:
+        return DumpOn[option.upper()]
+    sys.exit(f"Error: Invalid --dump-on option: {option}. Valid options: {DUMP_ON_OPTIONS_STRINGIFIED}")
```

Comment:
Can `Enum` use lowercased string values instead of `auto()`?

Reply:
It can, but I didn't want to duplicate the naming. And I can't find a way to automatically set it to be a lowercase version of the name.